docs/operations/observability.md

```diff
@@ -15,35 +15,35 @@ All components of Loki expose the following metrics:
 
 The Loki Distributors expose the following metrics:
 
-| Metric Name                                       | Metric Type | Description                                                 |
-| ------------------------------------------------- | ----------- | ----------------------------------------------------------- |
-| `loki_distributor_ingester_appends_total`         | Counter     | The total number of batch appends sent to ingesters.        |
-| `loki_distributor_ingester_append_failures_total` | Counter     | The total number of failed batch appends sent to ingesters. |
-| `loki_distributor_bytes_received_total`           | Counter     | The total number of uncompressed bytes received per tenant. |
-| `loki_distributor_lines_received_total`           | Counter     | The total number of lines received per tenant.              |
+| Metric Name                                       | Metric Type | Description                                                                                                             |
+| ------------------------------------------------- | ----------- | ----------------------------------------------------------------------------------------------------------------------- |
+| `loki_distributor_ingester_appends_total`         | Counter     | The total number of batch appends sent to ingesters.                                                                    |
+| `loki_distributor_ingester_append_failures_total` | Counter     | The total number of failed batch appends sent to ingesters.                                                             |
+| `loki_distributor_bytes_received_total`           | Counter     | The total number of uncompressed bytes received per tenant.                                                             |
+| `loki_distributor_lines_received_total`           | Counter     | The total number of log entries received per tenant (misleading name, as an entry can have more than one line of text). |
```

Comment:
Instead of misleading I would just explain that an entry can contains multiple line.

Reply:
Thanks for the feedback @cyriltovena -- I force-pushed with updated wording. Would appreciate if you could have another look. Thanks!